### PR TITLE
Feat/categorize

### PR DESCRIPTION
**Pull request by J4ninja**
## Type of Change

Feature Categorization Visualizations

## Context

#85 

## Changes Made

- Question Categorization Tab
- Question Distribution tab
- Question Collapsed Timeline tab
- Question Teacher Timeline
- Added my new headshot

## Screenshots
Screenshot 2024-03-26 at 12 54 30 PM
Screenshot 2024-03-26 at 12 54 35 PM
Screenshot 2024-03-26 at 12 54 43 PM
Screenshot 2024-03-26 at 12 54 49 PM

## Checklist

- [x] I have tested these changes locally
- [ ] I have updated the documentation, if applicable
- [ ] I have added unit tests, if applicable

### CODE REVIEW

File: frontend/src/images/johnheadshot2.jpeg
Comment (jhmejia) [inline frontend/src/images/johnheadshot2.jpeg:1]: pytest

File: frontend/src/components/Main/WhisperModel/QuestionDistribution.jsx
Comment (jhmejia) [inline frontend/src/components/Main/WhisperModel/QuestionDistribution.jsx:1]: Is it possible to put a link to the meaning of Costa's leveled questions? 

For example: https://www.lee.k12.nc.us/cms/lib03/NC01001912/Centricity/Domain/2165/Tutorial_Costa027.pdf

**Review by jhmejia — CHANGES_REQUESTED** [review]
just the one change, everything else looks okay

File: frontend/src/components/Main/WhisperModel/QuestionDistribution.jsx
Comment (J4ninja) [reply frontend/src/components/Main/WhisperModel/QuestionDistribution.jsx:1]: Added some titles with costa level to make it more clear

**Pull request merged**